packages/alfa-style/test/property/outline.spec.tsx

```diff
@@ -0,0 +1,103 @@
+import { test } from "@siteimprove/alfa-test";
+import { h } from "@siteimprove/alfa-dom/h";
+
+import { Device } from "@siteimprove/alfa-device";
+
+import { Style } from "../../src/style";
+
+const device = Device.standard();
+
+test("#computed() resolves `outline-offset: calc(1em + 2px)`", (t) => {
+  const element = <div style={{ outlineOffset: "calc(1em + 2px)" }} />;
+
+  const style = Style.from(element, device);
+
+  t.deepEqual(style.computed("outline-offset").toJSON(), {
+    value: {
+      type: "length",
+      value: 18,
+      unit: "px",
+    },
+    source: h.declaration("outline-offset", "calc(1em + 2px)").toJSON(),
+  });
+});
+
+test("#computed() rejects `outline-offset: calc(1 + 2)`", (t) => {
+  const element = <div style={{ outlineOffset: "calc(1 + 2)" }} />;
+
+  const style = Style.from(element, device);
+
+  t.deepEqual(style.computed("outline-offset").toJSON(), {
+    // initial value
```

Comment:
```suggestion
    // initial value
```
Clean up?

Reply:
That's on purpose. The calculation in that example is invalid for the property, so it defaults to its initial value.